tests/unittests/Foundation/ReferenceFoundation/TestNSArray.mm

```diff
@@ -172,6 +172,11 @@ static int objectIndexInArray(NSArray* array, int value, int startingFrom, int l
         @"If NSBinarySearchingInsertionIndex is specified and no other options provided NSArray returns any equal or one larger "
         @"index than any matching object’s index.");
 
+    auto anyIndexToInsertNineteen = objectIndexInArray(array, 19, 0, array.count, NSBinarySearchingInsertionIndex);
+    ASSERT_TRUE_MSG(
```

Comment:
Sure! I moved the tests regarding the NSBinarySearchingInsertionIndex option in the NSArrayTests.mm file.